app/models/like.rb

```diff
@@ -1,2 +1,12 @@
 class Like < ApplicationRecord
+  belongs_to :user, foreign_key: 'user_id'
+  belongs_to :post, foreign_key: 'post_id'
+  after_save :update_post_likes_counter
+  after_destroy :update_post_likes_counter
+
+  private
+
+  def update_post_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
### Optional

- [ ] You can use the same approach in the above optional suggestions to also update your likes count